test_graph_traversal.py

```diff
@@ -198,96 +139,151 @@ def cyclic_graph(node_list):
 # graph beginning at start. Return the full visited path when traversal is
 # complete.
 
-def test_dft_on_wide_at_root(wide_graph, node_list, wide_graph_map):
+def test_dft_on_wide_at_root(node_list, wide_graph, wide_graph_map):
     start = node_list[1]
-    actual = wide_graph.depth_first_traversal(start)
+    actualdwr = wide_graph.depth_first_traversal(start)
 
-    for node in enumerate(actual):
+    for node in enumerate(actualdwr):
         for item in wide_graph_map.iteritems():
             if node[1] in item[1]:
-                actual[node[0]] = item[0]
+                actualdwr[node[0]] = item[0]
 
     expected = [
         1, 2, 3, 4, 4, 3, 4, 4, 2, 3, 4,
         4, 3, 4, 4, 2, 3, 4, 4, 3, 4, 4
     ]
```

Comment:
This doesn't look right to me.  Are there actually that many 2s, 3s, and 4s in the graph? You should not be getting repeated values when you do either a DFT or a BFT